src/ucp/rndv/rndv.c

```diff
@@ -135,6 +135,23 @@ size_t ucp_rndv_rts_pack(ucp_request_t *sreq, ucp_rndv_rts_hdr_t *rndv_rts_hdr,
     return sizeof(*rndv_rts_hdr) + packed_rkey_size;
 }
 
+UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_rts, (self),
+                 uct_pending_req_t *self)
+{
+    ucp_request_t *sreq = ucs_container_of(self, ucp_request_t, send.uct);
+
+    if (sreq->flags & UCP_REQUEST_FLAG_SEND_TAG) {
```

Comment:
maybe avoid this branch and check both types of rts during FF?

Reply:
then need to include tag/tag_rndv.h file from UCP core code.
Also, it looks better for me to have a single function for RTs like we have for RTR

Reply:
IMO to have less changes and less branches, can compare to ucp_proto_progress_rndv/tag_rts and include tag_rndv.h/am.h

Reply:
done